docs/reference/cli-commands/start.md

```diff
@@ -22,7 +22,12 @@ any other services it depends on, in the correct order.
 
 ## How it works
 
-When starting a service, Pebble executes the service's `command`, and waits 1 second to ensure the command doesn't exit too quickly. Assuming the command doesn't exit within that time window, the start is considered successful, otherwise `pebble start` will exit with an error, regardless of the `on-failure` value.
+When starting a service, Pebble executes the service's `command`, and waits 1 second to ensure the command doesn't exit too quickly.
+
+- If the command doesn't exit within the 1 second window, the start is considered successful.
+- If the command exits within the 1 second window, Pebble retries the command after a backoff delay of half a second (default value). Within the 1 second window, Pebble applies the restart logic described in [Service auto-restart](/reference/service-auto-restart.md). You can configure the backoff delay and the auto-restart behaviour according to the service's exit code. For more information, see [Layer specification](/reference/layer-specification.md).
+
+If the service exits after each start attempt within the 1 second window, pebble start exits with an error, regardless of the on-failure value.
```

Comment:
Would it be possible to indent this sentence, as it's still talking about the failure case? Also suggesting a couple of edits for consistency:

```suggestion
    If the command exits after each start attempt within the 1 second window, `pebble start` exits with an error, regardless of the `on-failure` value.
```

Reply:
I'm having trouble understanding/parsing this sentence. It sounds like you're talking about the error that `pebble start` returns. Perhaps something like:

```
If one of the started services exits within the 1 second window, `pebble start` prints an appropriate error message and exits with an error.
```

Though I would add this to the end of the previous bullet point, now that that's a lot shorter.

We should check that it is really if *one* of the services exits within 1s, for example if you do `pebble start svc1 svc2 svc3`.

Reply:
Updated accordingly, and verified.

Layer:

```yaml
summary: a simple layer
services:
  svc1:
    override: replace
    command: bash -c "sleep 100"
  svc2:
    override: replace
    command: bash -c "exit 0"
```

Start:

```bash
$ ./pebble start svc1 svc2
error: cannot perform the following tasks:
- Start service "svc2" (service start attempt: exited quickly with code 0, will restart)
```

Changes and tasks:

```bash
./pebble changes
ID   Status  Spawn                     Ready                     Summary
1    Error   2 days ago, at 21:09 CST  2 days ago, at 21:09 CST  Autostart service "test"
2    Done    2 days ago, at 21:09 CST  2 days ago, at 21:09 CST  Stop all running services
3    Error   today at 10:44 CST        today at 10:44 CST        Start service "svc1" and 1 more
$ ./pebble tasks 3
Status  Spawn               Ready               Summary
Done    today at 10:44 CST  today at 10:44 CST  Start service "svc1"
Error   today at 10:44 CST  today at 10:44 CST  Start service "svc2"

......................................................................
Start service "svc2"

2024-12-04T10:44:41+08:00 INFO Most recent service output:

2024-12-04T10:44:41+08:00 ERROR service start attempt: exited quickly with code 0, will restart
```